src/model/element/entity.js

```diff
@@ -177,6 +177,25 @@ class Entity extends Element {
 
     return entity;
   }
+
+  toIndraTemplate(){
+    let name = this.name();
+
+    // if the name is not set it means that there is no info for the entity
+    // so that entity must not be included
+    if ( !name ) {
+      return null;
+    }
+
+    let xref = _.pick(this.association(), ['id', 'namespace']);
+    let validNamespaces = ['hgnc', 'chebi'];
+
+    if ( !_.includes( validNamespaces, xref.namespace ) ) {
+      xref = null;
+    }
+
+    return { name, xref };
```

Comment:
I suspect you want something output like this:

```
[
    {
      "name": "2'-3'-cGAMP",
      "xref": null
    },
    {
      "name": "SLC19A1",
      "xref": {
        "db": "HGNC",
        "id": "HGNC:10937"
      }
    }
  ]
```

Reply:
I guess the only difference is you have ``db`` instead of ``namespace``. Am I right? Actually it will only be internally used but I can update it as ``db``both here and where it is used if that is the case.

Reply:
I used `db` and `id` yes.